src/clincoded/static/components/variant_central/index.js

```diff
@@ -328,6 +337,150 @@ var VariantCurationHub = createReactClass({
+                            const experimentA = _.property(['entContent', 'Experiment', 'Number'])(a);
+                            const experimentB = _.property(['entContent', 'Experiment', 'Number'])(b);
+                            if (experimentA < experimentB) {
+                                return -1;
+                            }
+                            if (experimentA > experimentB) {
+                                return 1;
+                            }
+                            return 0;
+                        });
+                    } else {
+                        afisObject[pubmedId] = {
+                            statements: [record],
+                        };
+                    }
+                }
+            });
+            return afisObject;
+        };
+
+        // Fetches pubmed data for each source article
+        const getPubmedArticles = (afisObject) => {
+            const pubmedUrls = [];
+            const pubmedIds = [];
+            const pubmedIdKeys = Object.keys(afisObject);
+            if (pubmedIdKeys && pubmedIdKeys.length > 0) {
+                pubmedIdKeys.forEach(id => {
+                    pubmedIds.push(id);
+                });
+                const searchStr = '/search/?type=article&' + pubmedIds.map(pmid => 'pmid=' + pmid).join('&');
+                return this.getRestData(searchStr).then(articles => {
+                    if (articles['@graph'] && (articles['@graph'].length === pubmedIds.length)) {
+                        return articles['@graph'];
+                    }
+                    const foundArticles = articles['@graph'] ? articles['@graph'] : [];
+                    const missingPmids = _.difference(pubmedIds, articles['@graph'].map(article => article.pmid));
+                    missingPmids.forEach(id => {
+                        pubmedUrls.push(external_url_map['PubMedSearch'] + id);
+                    });
+                    return this.getRestDatasXml(pubmedUrls).then(xmls => {
+                        xmls.forEach(xml => {
+                            const data = parsePubmed(xml);
+                            if (data.pmid) {
+                                foundArticles.push(data);
+                            } else {
+                                throw { statusText: 'Missing PMID articles' };
+                            }
+                        });
+                        return foundArticles;
+                    }).catch(err => {
+                        throw err;
+                    });
+                });
+            } else {
+                this.setState({ loading_ldhFuncData: false });
+            }
+        };
+
+        // Sets pubmed articles in the @afisObject and replaces the AFIS with the completed @afisObject
+        const setPubmedArticles = (afisObject, articles) => {
+            if (!Array.isArray(articles)) {
+                return;
+            }
+            articles.forEach((article, articleIdx) => {
```

Comment:
Quick question! 

Is `articleIdx` meant to just be `articleId`? Or does it stand for " article index".